src/services/item/plugins/recycled/index.ts

```diff
@@ -28,11 +29,18 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
   // get own recycled items data
   fastify.get(
     '/recycled',
-    { schema: getRecycledItemDatas, preHandler: [isAuthenticated, matchOne(memberAccountRole)] },
-    async ({ user }) => {
+    {
+      schema: getOwnRecycledItemDatas,
+      preHandler: [isAuthenticated, matchOne(memberAccountRole)],
+    },
+    async ({ user, query }) => {
       const member = asDefined(user?.account);
       assertIsMember(member);
-      const result = await recycleBinService.getAll(member, buildRepositories());
+      const { page = 1, pageSize = ITEMS_PAGE_SIZE } = query;
+      const result = await recycleBinService.getOwn(member, buildRepositories(), {
+        page,
+        pageSize,
+      });
```

Comment:
I'm wondering if it is mandatory to define the default values here since you've already defined it in the schema ?
```suggestion
      const result = await recycleBinService.getOwn(member, buildRepositories(), query);
```